crates/sui-source-validation-service/README.md

```diff
@@ -65,3 +65,27 @@ For errors, or if the source code does not exist, an error encoded in JSON retur
 The URL parameters `address`, `module`, and `network` are required.
 
 Although not required, it is good practice to set the `X-Sui-Source-Validation-Version` header.
+
+## Hosted Service
+
+Mysten Labs maintains a backend service hosted at `https://source.mystenlabs.com` for verified packages. The following example usages are available via the API:
+
+- List current indexed sources via `curl 'https://source.mystenlabs.com/api/list'`. 
+  - For example, to see all verified sources on `mainnet`, query the `mainnet` member: `curl 'https://source.mystenlabs.com/api/list' --header 'X-Sui-Source-Validation-Version: 0.1'  | jq .mainnet`
+  
+- Get verified source for a `(address, module, network)` triple via a request like `curl 'https://source.mystenlabs.com/api?address=0x2&module=coin&network=mainnet' --header 'X-Sui-Source-Validation-Version: 0.1'`
+
+**Production Usage and Best Practices Tips*
+
+On occasion `https://source.mystenlabs.com` may return a `502` response or experience downtime. When using `https://source.mystenlabs` to determine source authenticity, it is recommended interface with the service in such a way that it can robustly handle a non-`200` response or downtime. For example, when a non-`200` response is received, a tab showing verified source may temporarily show a message like `"Not available at this time"` when used in an explorer. Once the source verification resumes, requests will continue to allow clients to show source as normal.
```

Comment:
```suggestion
On occasion `https://source.mystenlabs.com` may return a `502` response or experience downtime. When using `https://source.mystenlabs.com` to determine source authenticity, it is recommended interface with the service in such a way that it can robustly handle a non-`200` response or downtime. For example, when a non-`200` response is received, a tab showing verified source may temporarily show a message like `"Not available at this time"` when used in an explorer. Once the source verification resumes, requests will continue to allow clients to show source as normal.
```